tests/beta/interfaces/test_tape_autograd.py

```diff
@@ -732,3 +730,559 @@ def cost(x, device):
         x = np.array(0.543, requires_grad=True)
         res = cost(x, device=dev)
         assert res.shape == (2, 10)
+
+
+@pytest.mark.parametrize(
+    "dev_name,diff_method",
+    [["default.qubit", "finite-diff"], ["default.qubit.autograd", "backprop"]],
+)
+class TestQNode:
+    """Same tests as above, but this time via the QNode interface!"""

```

Comment:
Would it be worth splitting this part into a `test_tape_autograd_integration.py` or `test_tape_autograd_qnode` file? (~1300 lines and could grow potentially)

Reply:
Good idea!